cli/batch/sql.go

```diff
@@ -191,9 +207,10 @@ func buildBulkInsert(tbl *dax.Table, fields []*dax.Field, ids []interface{}, row
 	// bulk insert as one line in the NDJSON payload. We re-use the map for each
 	// row.
 	m := make(map[string]interface{})
+
 	for i := range rows {
 		// Write the ID value.
-		m[string(dax.PrimaryKeyFieldName)] = ids[i]
+		m[string(dax.PrimaryKeyFieldName)] = fmt.Sprintf(fmtStr, ids[i])
```

Comment:
I didn't spend a lot of time here. But in some case IDK gives us a `[]byte` if the table is keyed. And then it writes the wrong thing to FeatureBase. Here is a test showing that. 
```
Error:          Not equal: 
                                expected: []interface {}{[]interface {}{"1", "a", 20, []interface {}{"hob1", "hob2"}}, []interface {}{"2", "b", 21, []interface {}{"hob2", "hob3"}}, []interface {}{"3", "c", 22, []interface {}{"hob3", "hob4"}}, []interface {}{"4", "d", 23, []interface {}{"hob4", "hob5"}}, []interface {}{"5", "e", 24, []interface {}{"hob5", "hob6"}}, []interface {}{"6", "f", 26, []interface {}{"hob6", "hob7"}}}
                                actual  : []interface {}{[]interface {}{"MQ==", "a", 20, []interface {}{"hob1", "hob2"}}, []interface {}{"Mg==", "b", 21, []interface {}{"hob2", "hob3"}}, []interface {}{"Mw==", "c", 22, []interface {}{"hob3", "hob4"}}, []interface {}{"NA==", "d", 23, []interface {}{"hob4", "hob5"}}, []interface {}{"NQ==", "e", 24, []interface {}{"hob5", "hob6"}}, []interface {}{"Ng==", "f", 26, []interface {}{"hob6", "hob7"}}}
```
I can think about how to avoid this.

Reply:
I decided to change this so it's only formatting with keyed indexes to handle the []byte vs string case.